common/src/main/java/org/apache/celeborn/common/protocol/message/StatusCode.java

```diff
@@ -68,7 +68,11 @@ public enum StatusCode {
   REGION_FINISH_FAIL_SLAVE(36),
   REGION_FINISH_FAIL_MASTER(37),
 
-  PUSH_DATA_TIMEOUT(38);
+  PUSH_DATA_CONNECT_FAIL_MASTER(38),
```

Comment:
```PUSH_DATA_CONNECT_FAIL_``` and ```PUSH_DATA_CONNECTION_FAIL_``` is quite confusing. IMO we can rename to ```PUSH_DATA_CREATE_CONNECTION_FAIL_``` and ```PUSH_DATA_CONNECTION_EXCEPTION_```

Reply:
Done